### ISSUE TITLE
Chat overlapping with vote screen

### ISSUE THREAD

**Issue by Dragonisser** [CriticalFlaw/TF2HUD.Editor#206]
Version: v2023.0730
Screen resolution: 2560x1600 (16:10)

- Using the option for the lower chat. 

I assume that's the issue why its overlapping. Is there any way to move the vote screen further up instead?

![image](https://github.com/CriticalFlaw/flawhud/assets/4233773/c72f2229-48af-4d00-a086-7ad820c8ac8f)

<img width="316" alt="Screenshot_1" src="https://github.com/CriticalFlaw/flawhud/assets/4233773/2b9d3674-e9fa-4c72-a988-22dee8c1fad6">
<img width="433" alt="Screenshot_2" src="https://github.com/CriticalFlaw/flawhud/assets/4233773/921dfcb0-177d-4e47-8afa-185eba6f9d49">

**Comment by Dragonisser** [comment]
Right, i forgot that buildings are up there and achievements down there (never had those on my hud). Ill guess i have to stick with that then. Thanks.

**Comment by CriticalFlaw** [comment]
16:10 specific customizations could be added. But I'd need a 16:10 monitor to really create it. Unless someone else wants to do it

**Comment by Dragonisser** [comment]
I could possible help testing them or try it? Depends on what would needed to be changed (this for example https://github.com/CriticalFlaw/flawhud/issues/358)

**Comment by CriticalFlaw** [comment]
It'd be easier if someone submitted a pull request with all the changes for 16:10 that could then either be merged in if they don't cause any issues for other resolutions or made into an editor option.
